docs/source/fx.rst

```diff
@@ -298,6 +290,245 @@ Miscellanea
+    print(traced)
+
+    # Print the internal Graph. This representation returns:
+    """
+    graph(x, y):
+        %add_1 : [#users=1] = call_function[target=<built-in function add>](args = (%x, %y), kwargs = {})
+        return add_1
+    """
+    print(traced.graph)
+
+    # Print a tabular representation of the internal Graph. This gives us:
+    """
+    opcode         name    target                   args      kwargs
+    -------------  ------  -----------------------  --------  --------
+    placeholder    x       x                        ()        {}
+    placeholder    y       y                        ()        {}
+    call_function  add_1   <built-in function add>  (x, y)    {}
+    """
+    traced.graph.print_tabular()
+
+Using the utility functions above, we can compare our traced Module
+before and after we've apply our transformations. Sometimes, a
+simple visual comparison is enough to trace down a bug. If it's still
+not clear what's going wrong, a debugger like ``pdb`` can be a good
+next step.
+
+Going off of the example above, consider the following code:
+
+::
+
+    # Sample user-defined function
+    def transform_graph(gm: GraphModule) -> None:
+
+        # Get the Graph from our traced Module
+        g = gm.graph
+
+        """
+        Transformations on `g` go here
+        """
+
+        # Recompile the GraphModule. This must be called after editing
+        # the Graph `g`, otherwise the generated code will still reflect
+        # the old Graph before any transforms
+        gm.recompile()
+
+    # Transform the Graph
+    transform_graph(traced)
+
+    # Print the new code after our transforms. Check to see if it was
+    # what we expected
+    print(traced)
+
+Using the above example, let’s say that the call to ``print(traced)``
+showed us that there was an error in our transforms. We want to find
+what goes wrong using a debugger. We start a ``pdb`` session. We can see
+what’s happening during the transform by breaking on
+``transform_graph(traced)``, then pressing ``s`` to “step into” the call
+to ``transform_graph(traced)``.
+
+We may also have good luck by editing the ``print_tabular`` method to print
+different attributes of the Nodes in the Graph. (For example, we might
+want to see the Node’s ``input_nodes`` and ``users``.)
+
+.. _Available Debuggers:
+
+Available Debuggers
+^^^^^^^^^^^^^^^^^^^^^^
+
+The most common Python debugger is
+`pdb <https://docs.python.org/3/library/pdb.html>`__. You can start
+your program in “debug mode” with ``pdb`` by typing
+``python -m pdb FILENAME.py`` into the command line, where ``FILENAME``
+is the name of the file you want to debug. After that, you can use the
+``pdb`` `debugger
+commands <https://docs.python.org/3/library/pdb.html#debugger-commands>`__
+to move through your running program stepwise. It’s common to set a
+breakpoint (``b LINE-NUMBER``) when you start ``pdb`` then call ``c`` to
+run the program until that point. This prevents you from having to step
```

Comment:
`set_trace()` is used in the above examples so probably a good idea to explicitly mention it